automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoSpec.scala

```diff
@@ -177,6 +191,12 @@ class LeonardoSpec extends FreeSpec with Matchers with Eventually with ParallelT
       }
     }
 
+    "should error on cluster create and delete the cluster" in {
+      withNewErroredCluster(project) { _ =>
+        // no-op; just verify that it launches
```

Comment:
confused. shouldn't there be assertions in here?

Reply:
The assertions are inside of the withNewErroredCluster

Reply:
Ah ok, thanks.

Reply:
Heh Rob didn't like it when I started this pattern either.  :)

Should change the text to "verify that it errors" or similar